Zend/zend_ini.c

```diff
@@ -30,6 +30,8 @@ static HashTable *registered_zend_ini_directives;
 #define NO_VALUE_PLAINTEXT		"no value"
 #define NO_VALUE_HTML			"<i>no value</i>"
 
+#define ZEND_IS_WHITESPACE(c) (((c) == ' ') || ((c) == '\t') || ((c) == '\n') || ((c) == '\r') || ((c) == '\v') || ((c) == '\f'))
```

Comment:
Make this an inlined function? It's easy to mess up if using the macro like this: `ZEND_IS_WHITESPACE(*x++);`